### DISCUSSION TITLE
保護鯨魚—AI 浮標

### DISCUSSION THREAD

**Discussion by Zoe06060606** [1101-GS4719-C/Python-Community-GS4719-C#798] [General]
據世界永續組織估計，每年皆有18,000至25,000頭鯨魚因船隻撞擊而喪命。為減少此事件發生，智利政府與MERI基金會合作，推動「Blue Boat Initiative」的計畫，在智利南部的科爾科瓦多灣外海設置了「AI浮標」，以警告附近船隻有鯨魚的存在。
   AI浮標整合數種水下聽音設備，並以 AI分析音頻，由叫聲推測鯨魚種類及分佈位置。浮標同時會持續監測水中的含氧量及溫度，協助保育單位了解海洋狀況之長期變化。

看到世界永續組織的數據我感到震驚，沒想到會有那麼多鯨魚遭到船隻撞擊死亡，希望這個計畫能持續進行到佈滿鯨魚會遷徙的所有路線，以減少鯨魚的死亡率。
[AI浮標](https://tw.news.yahoo.com/智利政府與民間組織在海中放置-ai-浮標-來保護鯨魚-110028294.html)

**Comment by bonny123865** [comment]
我看到那個數據也被嚇到!
希望這個AI浮標能發揮好的作用，保護鯨魚們的生命。
我覺得這項技術之後除了鯨魚之外，大型海生動物如果也能偵測並繞道，那對於海底生物的生存環境就更加友善了!

**Comment by moonnight3503** [comment]
確實，如果能有個AI浮標保護鯨魚著實不錯，可藉由分析鯨魚遷徙路線來更改航線，並且同時能夠偵測海洋環境的資訊，真是一舉兩得的方法。

**Comment by AnAnistired** [comment]
看起來的確對保護鯨魚很實用，但如何大量投放以及更新設備又是一筆支出，這種行動如果單純由社會公益團體主導，沒有政府部門的幫助，實在很難執行。

**Comment by evanchou1204** [comment]
我也覺得成本可能會是很大的問題，除了公益團體之外，可能需要更多人願意花大錢去保護鯨魚😢

**Comment by Chiway1229** [comment]
不過這種環境保育的提案，在台灣推行起來可能又會困難重重了呢...

**Comment by ABCDEFGHIJKLMNO12345678** [comment]
雖然概念很好 但經費也是不小的問題 沒有錢萬萬不能啊

**Comment by delia07152** [comment]
謝謝分享，沒想到鯨魚會因為撞到船隻而喪命，更沒想到每年的案例竟然那麼多，我剛剛查資料發現鯨魚可以協助捕捉二氧化碳進而減緩全球暖化，希望可以藉由智慧浮標讓鯨魚免受船隻的撞擊，大家一起保護鯨魚，鯨魚也保護我們!

**Comment by Pietra1226** [comment]
如果能藉由科技的力量讓自然跟人類取得平衡是最好的，就怕越來越發達我們會忘我地一直侵害到其他生物的生存空間。

**Comment by manipulateed** [comment]
此舉能有效保護海洋生物而不局限於鯨魚，且能更好掌控航海路線也能降低航海人員航行的風險，但工程應該可謂艱難，期許能慢慢突破瓶頸

**Comment by Evan0979** [comment]
撞船而死,好可憐QQ,希望AI 浮標能幫到他們

**Comment by StevenWanglolz** [comment]
每年有18,000到25,000頭鯨魚被撞死!!!那麼大之隻ㄟ，好扯! 希望這些AI浮標能幫助保育單位了解鯨魚的情況，然後減少這種意外事故 :(

**Comment by CPCpyt** [comment]
在近期環境保護真的是非常大的問題，
希望這個技術能夠趕快普及保護大自然！
